pkg/infra/tracing/tracing.go

```diff
@@ -95,6 +95,9 @@ type Tracer interface {
 	// Both the context and span must be derived from the same call to
 	// [Tracer.Start].
 	Inject(context.Context, http.Header, Span)
+
+	// OtelTracer returns the trace.Tracer if available or nil.
+	OtelTracer() trace.Tracer
```

Comment:
This is a somewhat ugly hack.

@grafana/backend-platform any plans removing this Tracer interface and default to the otel Tracer interface now when otel is the only supported tracing library? Alternatively, make the Grafana Tracer interface implement the otel Tracer interface?